### DISCUSSION TITLE
FLEX Calc

### DISCUSSION THREAD

**Discussion by frogmonkey2** [flybywiresim/aircraft#5194] [Ideas]
We need a functional Flex calculation tool in the Flypad. Preferably one that can gather their DATA by airport IE: KLAX - KSFO. If it's already mentioned or in the works, forgive. THX

**Comment by tracernz** [comment]
We need somebody to collect the data and write the code for it. 😄

**Reply by frogmonkey2** [reply]
well I'm sure some smart person will eventually be able to code us one😎

**Reply by frogmonkey2** [reply]
honestly speaking as a clueless man... could something like this be forked to the project? in the Flypad?...

**Reply by tracernz** [reply]
We'll come up with something when the time is right. We're still working on getting the MSFS SDK, and our own model, improved to the point where the perf data would be meaningful and accurate.

**Comment by frogmonkey2** [comment]
there's a 'new' one out there that's basically a XL spreadsheet. you need to find out the airports weather yourself tho

**Reply by cloudynetwork** [reply]
Where can this be found please?
